hamocc/mo_read_fedep.F90

```diff
@@ -132,12 +132,34 @@ subroutine get_fedep(kpie,kpje,kplmon,dust)
     !  J.Schwinger            *NORCE Climate, Bergen*       2020-05-19
     !***********************************************************************************************
 
+    use mod_xc             , only: nbdy
+    use mod_output_forcing , only: output_forcing
+
     integer, intent(in)  :: kpie             ! 1st dimension of model grid
     integer, intent(in)  :: kpje             ! 2nd dimension of model grid
     integer, intent(in)  :: kplmon           ! current month.
     real,    intent(out) :: dust(kpie,kpje)  ! dust flux for current month
 
-    dust = dustflx(:,:,kplmon)
+    integer :: i,j
+    logical :: debug = .true.
+    logical :: first_time = .true.
+    real, allocatable :: dust_out(:,:)
+
+    dust(:,:) = dustflx(:,:,kplmon)
+
+    if (debug) then
```

Comment:
Hi @mvertens , do you want to keep the debug flag here at the end or is it something temporary? If the first, I wonder, if we should use our common hamocc debug flag(s) instead.

Reply:
It's up to you. Do you always want to see the netcdf file when the model starts?